assembl/static2/js/app/graphql/mutations/updateIdeas.graphql

```diff
@@ -0,0 +1,44 @@
+mutation updateIdeas($discussionPhaseId: Int!, $ideas: [IdeaInput]!) {
+  updateIdeas(discussionPhaseId: $discussionPhaseId, ideas: $ideas) {
+    rootIdea {
+      children {
```

Comment:
maybe remove some of result part or add a comment to warn us that the result is not used and thus don't have to be recursive

Reply:
ok, you need the result for unit tests at least

Reply:
Indeed, the result is not used in reality but it's currently used in tests

Reply:
I now return the ideas fragment which is the same as ThematicsQuery.